core/dungeon-masters-guide/items/items-weapons.xml

```diff
@@ -538,6 +538,210 @@
+			</ul>
+			<p class="indent"><i><b>Conjure Earth Elemental.</b></i> If you are holding the axe, you can use your action to cast the <i>conjure elemental</i> spell from it, summoning an earth elemental. You can't use this property again until the next dawn.</p>
+			<p class="indent"><i><b>Travel the Depths.</b></i> You can use an action to touch the axe to a fixed piece of dwarven stonework and cast the <i>teleport</i> spell from the axe. If your intended destination is underground, there is no chance of a mishap or arriving somewhere unexpected. You can't use this property again until 3 days have passed.</p>
+			<p class="indent"><i><b>Curse.</b></i> The axe bears a curse that affects any non-dwarf that becomes attuned to it. Even if the attunement ends, the curse remains. With each passing day, the creature's physical appearance and stature become more dwarflike. After seven days, the creature looks like a typical dwarf, but the creature neither loses its racial traits nor gains the racial traits of a dwarf. The physical changes wrought by the axe aren't considered magical in nature (and therefore can't be dispelled), but they can be undone by any effect that removes a curse, such as a <i>greater restoration</i> or <i>remove curse</i> spell.</p>
+			<p class="indent"><i><b>Destroying the Axe.</b></i> The only way to destroy the axe is to melt it down in the <em>Earthheart Forge</em>, where it was created. It must remain in the burning forge for fifty years before it finally succumbs to the fire and is consumed.</p>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_ARTIFACT_TABLE" />
+			</div>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_MAGIC_ITEM_BELT_OF_DWARVENKIND" />
+			</div>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_MAGIC_ITEM_DWARVEN_THROWER" />
+			</div>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_MAGIC_ITEM_SWORD_OF_SHARPNESS" />
+			</div>
+		</description>
+		<setters>
+			<set name="keywords"></set>
+			<set name="category">Magic Weapons</set>
+			<set name="type" addition="battleaxe">Weapon</set>
+			<set name="rarity">Artifact</set>
+			<set name="enhancement">3</set>
+			<set name="attunement">true</set>
+			<set name="weapon">Battleaxe</set>
+		</setters>
+		<rules>
+			<stat name="constitution" value="2" requirements="ID_SRD_RACE_DWARF" />
+			<grant type="Language" id="ID_LANGUAGE_DWARVISH" requirements="ID_SRD_RACE_DWARF" />
+			<grant type="Vision" id="ID_VISION_DARKVISION" requirements="ID_SRD_RACE_DWARF"/>
+			<grant type="Condition" id="ID_INTERNAL_CONDITION_DAMAGE_IMMUNITY_POISON" requirements="ID_SRD_RACE_DWARF"/>
+			<grant type="Proficiency" id="ID_PROFICIENCY_TOOL_PROFICIENCY_BREWERS_SUPPLIES" requirements="ID_SRD_RACE_DWARF"/>
+			<grant type="Proficiency" id="ID_PROFICIENCY_TOOL_PROFICIENCY_SMITHS_TOOLS" requirements="ID_SRD_RACE_DWARF"/>
+			<grant type="Proficiency" id="ID_PROFICIENCY_TOOL_PROFICIENCY_MASONS_TOOLS" requirements="ID_SRD_RACE_DWARF"/>
```

Comment:
I don't think these are correct, i believe even if you're a Human you can receive full benefits of the item, except for the Blessings of Moradin.
```suggestion
			<stat name="constitution" value="2" />
			<grant type="Language" id="ID_LANGUAGE_DWARVISH" requirements="!ID_SRD_RACE_DWARF" />
			<grant type="Vision" id="ID_VISION_DARKVISION" requirements="!ID_SRD_RACE_DWARF"/>
			<grant type="Condition" id="ID_INTERNAL_CONDITION_DAMAGE_IMMUNITY_POISON" requirements="ID_SRD_RACE_DWARF"/>
			<stat name="darkvision:range" value="60" requirements="ID_SRD_RACE_DWARF" />
			<grant type="Proficiency" id="ID_PROFICIENCY_TOOL_PROFICIENCY_BREWERS_SUPPLIES" requirements="ID_SRD_RACE_DWARF"/>
			<grant type="Proficiency" id="ID_PROFICIENCY_TOOL_PROFICIENCY_SMITHS_TOOLS" requirements="ID_SRD_RACE_DWARF"/>
			<grant type="Proficiency" id="ID_PROFICIENCY_TOOL_PROFICIENCY_MASONS_TOOLS" requirements="ID_SRD_RACE_DWARF"/>
```